.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish to npm
+
+on:
+  push:
+    branches:
+      - "main"
+    paths-ignore:
+      - "docs/**"
+
+# Ensure only one workflow/job is publishing to npm at a time
+concurrency: publish-to-npm
+
+jobs:
+  release:
+    if: github.repository == 'latticexyz/mud'
+    uses: "./.github/workflows/release.yml"
+    secrets: inherit
+
+  snapshot:
+    if: github.repository == 'latticexyz/mud'
+    needs: [release]
```

Comment:
not sure if the conditional is required here and below if `needs` is not met by earlier conditional, but added it here anyway